tests/requirements/base.txt

```diff
@@ -6,6 +6,8 @@ jinja2 >= 2.7
 numpy
 Pillow
 PyYAML
+# Remove if it ends up being possible to build libmemcached on the Windows Jenkins box.
```

Comment:
Oh, I see. Then I guess I'll add something like this to the settings file if it seems reasonable:
```
if sys.platform.startswith('win'):
   del CACHES['pylibmc']
```